community/merchants/index.md

```diff
@@ -99,10 +99,11 @@ permalink: /community/merchants/index.html
         <h3>Swappers</h3>
         <p>{% t merchants.swappersp %}</p>
         <ul class="logo">
+            <li><a href="https://https://swapspace.co/">SwapSpace.co</a> - instant cryptocurrency exchange aggregator</li>
```

Comment:
```suggestion
            <li><a href="https://swapspace.co/">SwapSpace.co</a> - instant cryptocurrency exchange aggregator</li>
```

I don't know if we currently list new exchanges, but in any case, the link is incorrect.